tests/analyzer/sourcecode/clipboard-paste.py

```diff
@@ -0,0 +1,34 @@
+""" Tests for clipboard-paste rule
+
+    RULEID cases:
+      - KEWK malware
+    No Clear OK cases
+"""
+
+""" RULEID: KEWK malware
+"""
+# ruleid: clipboard-paste
+clipboard_data = pyperclip.paste()
+if re.search('^[13][a-km-zA-HJ-NP-21-9]{25, 34}$', clipboard_data):
+    if clipboard_data not in [self.address_btc, self.address_eth, self.address_xchain]:
+        if self.address_btc != "none":
+            pyperclip.copy(self.address_btc)
+            # ruleid: clipboard-paste
+            pyperclip.paste()
```

Comment:
any clue why the malware uses pyperclip.paste() here?

Reply:
Oh I don't think it has any impact. I'll remove it